Telerik.JustMock/Core/MockingUtil.PrivateMethods.cs

```diff
@@ -330,6 +370,17 @@ public static MethodInfo GetMethodWithLocalFunction(Type type, string methodName
 			{
 				throw new MissingMemberException(MockingUtil.BuildMissingMethodMessage(type, null, methodName));
 			}
+
+            if(method.ContainsGenericParameters && (methodGenericTypes == null || methodGenericTypes.Length == 0))
+            {
+                throw new MissingMemberException(MockingUtil.BuildMissingMethodMessage(type, null, methodName));
```

Comment:
Please make sure that the message in the exception is very clear why the member is missing. Think of answering the client question: "I am passing the correct arguments. I can see that the method is there. Why this exception says it is missing?".